src/microbe_stage/MicrobeHUD.cs

```diff
@@ -1076,4 +1106,42 @@ private void OnAbilitiesHotBarDisplayChanged(bool displayed)
     {
         hotBar.Visible = displayed;
     }
+
+    private class HoveredCompoundControl : HBoxContainer
+    {
+        private readonly Label compoundName = new Label();
+        private readonly Label compoundValue = new Label();
+
+        public HoveredCompoundControl(Compound compound)
+        {
+            MouseFilter = MouseFilterEnum.Ignore;
+            Compound = compound;
+            TextureRect compoundIcon = GUICommon.Instance.CreateCompoundIcon(compound.InternalName, 20, 20);
+            compoundName.SizeFlagsHorizontal = (int)SizeFlags.ExpandFill;
+            compoundName.Text = Compound.Name;
+            AddChild(compoundIcon);
+            AddChild(compoundName);
+            AddChild(compoundValue);
```

Comment:
For consistency with other code, these also should be put in _Ready (as well as the MouseFilter line)